src/cache_cf.cc

```diff
@@ -1588,6 +1590,12 @@ static void
 parse_acl_address(Acl::Address ** head)
```

Comment:
Please do not forget to update the dump_acl_address() function. It needs to print tcp_outgoing_addr configuration.

Reply:
I've done that in my latest changes, but it's worth noting that when I tried searching from my IDE, I couldn't find any instances where `dump_acl_address()` is used. I'm curious to know how that function would get invoked.